erpnext/stock/doctype/quality_inspection/quality_inspection.js

```diff
@@ -60,6 +60,18 @@ frappe.ui.form.on("Quality Inspection", {
 	},
 })
 
+frappe.ui.form.on("Quality Inspection Reading", {
+	status: function (frm, cdt, cdn) {
+		let row = locals[cdt][cdn];
+		if (row.status === "Rejected") {
+			frappe.confirm(__("'" + row.specification + "' Parameter in Readings has 'Rejected' status, Are you sure you want to proceed?"),
+				() => { frm.set_value("status", row.status); },
+				() => { }
```

Comment:
If nothing needs to happen, you should `frappe.reload_doc()` so that the value moves back to Accepted.